### PR TITLE
Add 'Random mismatches' button

### PR DESCRIPTION
**Pull request by guergana**
Bug: [T302289](https://phabricator.wikimedia.org/T302289)

### CODE REVIEW

**Comment by guergana** [comment]
Would it make sense to add a browser test for this?

**Review by lucaswerkmeister — APPROVED** [review]
Looks good and seems to work fine.

> Would it make sense to add a browser test for this?

Probably, if it’s not too difficult…

File: tests/Browser/RandomButtonTest.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+class RandomButtonTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+
+    public function test_clicking_random_mismatches_button_redirects_and_shows_results_when_mismatches_available()
+    {
+        $import = ImportMeta::factory()
+        ->for(User::factory()->uploader())
+        ->create();
```
Comment (lucaswerkmeister) [inline tests/Browser/RandomButtonTest.php:21]:
```suggestion
            ->for(User::factory()->uploader())
            ->create();
```

**Review by lucaswerkmeister — COMMENTED** [review]
LGTM, just one indentation issue

**Review by lucaswerkmeister — APPROVED** [review]

**Pull request merged**